data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -368,16 +370,30 @@ fi
 
 echo "Starting cluster-bootstrap..."
 
-if [ ! -f cb-bootstrap.done ]
-then
-    bootkube_podman_run \
+run_cluster_bootstrap() {
+	bootkube_podman_run \
         --rm \
         --volume "$PWD:/assets:z" \
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
         start --tear-down-early=false --asset-dir=/assets --required-pods="${REQUIRED_PODS}"
+}
+
+gather_bootstrap_on_failure() {
```

Comment:
Is this function useful any more? Or can we call `cluster_bootstrap_gather` directly instead?

Reply:
Yeah you're right. Maybe I need to slow down the changes and think. Sorry :-)